mavros/src/plugins/waypoint.cpp

```diff
@@ -736,6 +739,14 @@ class WaypointPlugin : public plugin::PluginBase {
 		if (req.start_index)
 		{
 			//Partial Waypoint update
+
+            if(!enable_partial_push)
+            {
+                ROS_WARN_NAMED("wp","Partial Push not enabled. (Only supported on APM)");
+                res.success = false;
+                res.wp_transfered = 0;
+                return true;
+            }
```

Comment:
don't use spaces. Use tab for indentation (equivalent to 8 spaces).

Reply:
Or just run the uncrustify tool on the file (`uncrustify -c ${ROS_WORKSPACE}/mavros/mavros/tools/uncrustify-cpp.cfg --replace --no-backup <path/to/file.ext>`)